### DISCUSSION TITLE
HelmRelease updated values.yaml

### DISCUSSION THREAD

**Discussion by Sleepy-GH** [fluxcd/flux2#504] [Q&A] [answered]
Hi folks,

I'm really enjoying Flux v2 and I'm building up a nice cluster with it right now!
I was just wondering if there's an easy way to update a helm release when you update the values.yaml

My current way of updating is by just deleting the entire helm release, wait for all the resources to be clear and reconciling the HelmRelease with the new values.yaml, but I feel like there should be a more elegant way of going about it.

Any thoughts or suggestions?

**Comment by hiddeco** [comment, marked as answer]
The new values should be automatically detected on the next reconciliation run when you update them in the `spec.values` of the `HelmRelease` or in any of the referenced `spec.valuesFrom` resources.

**Reply by Sleepy-GH** [reply]
Hmm. When I added some values that should have updated a ConfigMap nothing happened though. I even completely deleted the ConfigMap and ran a flux reconcile on the helmrelease, but it didn't get recreated.

**Reply by hiddeco** [reply]
Can you please share a pseudo-copy of your `HelmRelease` and a precise description of the actions you performed.

**Reply by Sleepy-GH** [reply]
No need anymore, it works like a charm. I just had a typo in my values.yaml so my changes didn't get picked up.
Just had to update my values.yaml again and it works perfectly 😃

**Comment by Sleepy-GH** [comment]
Just upgraded Flux to v0.3.0 to make sure that it wasn't because I was running an older version, but I'm still running into this

**Comment by Sleepy-GH** [comment]
Nevermind, you're right. I had a mistake in my values.yaml... Sometimes typing is hard.
